controllers/suite_test.go

```diff
@@ -71,10 +80,35 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(k8sClient).NotTo(BeNil())
 
-}, 60)
+	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
+		Scheme:             scheme.Scheme,
+		MetricsBindAddress: "0",
+	})
+	Expect(err).ToNot(HaveOccurred())
+
+	// comment in when moving to "normal" testEnv test
```

Comment:
comment needed?

Reply:
well... at least for me it has a meaning 😉 Since it doesn't hurt, I leave it